tests/ui/traits/new-solver/pointer-like.rs

```diff
@@ -0,0 +1,12 @@
+#![feature(pointer_like_trait)]
```

Comment:
Looks like this test got duplicated on accident.

Also, since this is in the `new-solver` folder, I think this is supposed to have `// compile-flags: -Ztrait-solver=next`?

Reply:
lol yes